amphibia-reptilia.csl

```diff
@@ -58,13 +63,13 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-min="7" et-al-use-first="6">
+  <bibliography delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all" entry-spacing="2" line-spacing="2" hanging-indent="true">
     <sort>
       <key macro="author"/>
       <key macro="issued"/>
       <key variable="title"/>
     </sort>
-    <layout suffix=".">
+    <layout prefix="•&#9;" suffix=".">
```

Comment:
@RAWes, I'm pretty sure the journal doesn't actually want bullet points. See e.g. the bibliography in the PDF at http://dx.doi.org/10.1163/15685381-00002859.